packages/system/src/components/skeleton.css.ts

```diff
@@ -0,0 +1,37 @@
+import { style, keyframes, styleVariants } from "@vanilla-extract/css"
```

Comment:
Unless these skeleton components are used outside of the comparison table, I think it's best to move these inside `components/comparison-table/components`.